app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RideshareRails</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <%= link_to "Passengers", passengers_path %>
+          </li>
+          <li>
+            <%= link_to "Add a passenger", new_passenger_path %>
+          </li>
+          <li>
+            <%= link_to "Drivers", drivers_path %>
+          </li>
+          <li>
+            <%= link_to "Add a driver", new_driver_path %>
+          </li>
+          <li>
+            <%= link_to "Trips", trips_path %>
+          </li>
+          <li>
+            <%= link_to "Add a trip", new_trip_path %>
```

Comment:
Does this really go here?  Shouldn't you only need to create a trip on the passenger's show page?